sdk/src/cameras/chicony-006/Chicony_EEPROM.h

```diff
@@ -0,0 +1,41 @@
+#include <algorithm>
+#include <array>
+#include <memory>
+
+class Chicony_EEPROM
```

Comment:
This file seems to have some issues regarding our coding style that is not touched with the clang-format check:
1. the class Name with a `_`, I'm not sure if we are using this anywhere else, so this can become `ChiconyEEPROM` or even `ChiconyEeprom`
2. missing include guards
3. method names seem to be in Pascal Case, while we are using Camel Case in the rest of the sdk, and some even have `_`.